pkg/scheduler/core/generic_scheduler.go

```diff
@@ -190,19 +201,29 @@ func (g *genericScheduler) Schedule(pod *v1.Pod, nodeLister algorithm.NodeLister
 	// When only one node after predicate, just use it.
 	if len(filteredNodes) == 1 {
 		metrics.SchedulingAlgorithmPriorityEvaluationDuration.Observe(metrics.SinceInMicroseconds(startPriorityEvalTime))
-		return filteredNodes[0].Name, nil
+		return ScheduleResult{
+			SuggestedHost:  filteredNodes[0].Name,
+			EvaluatedNodes: 1,
```

Comment:
This is after predicates have run, so how do we know the number of nodes evaluated?

Reply:
A good catch.
I think it is len(filteredNodes) + len(failedPredicateMap).